src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -429,29 +438,38 @@ private void SetupScene()
             ShowShadows = false;
             
             // setup lighting            
-            //AmbientLightColor = new Color4(0.3f, 0.3f, 0.3f, 1.0f);
             AmbientLightColor = new Color4(0.0f, 0.0f, 0.0f, 1.0f);
 
             DirectionalLightColor = new Color4(0.9f, 0.9f, 0.9f, 1.0f);
             DirectionalLightDirection = new Vector3(-0.5f, -1.0f, 0.0f);
             
-            //FillLightColor = new Color4(new Vector4(0.2f, 0.2f, 0.2f, 1.0f));
             FillLightColor = new Color4(new Vector4(0.0f, 0.0f, 0.0f, 1.0f));
             FillLightDirection = new Vector3(0.5f, 1.0f, 0f);
 
-            var matColor = (System.Windows.Media.Color)ColorConverter.ConvertFromString("#efede4");
+            var matColor = (System.Windows.Media.Color)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["MaterialColor"];
+            materialColor = new Color4(matColor.R/255.0f, matColor.G/255.0f, matColor.B/255.0f, matColor.A/255.0f);
             RenderTechnique = Techniques.RenderPhong;
             WhiteMaterial = new PhongMaterial
             {
                 Name = "White",
                 AmbientColor = PhongMaterials.ToColor(0.1, 0.1, 0.1, 1.0),
-                //DiffuseColor = PhongMaterials.ToColor(0.992157, 0.992157, 0.992157, 1.0),
-                DiffuseColor = PhongMaterials.ToColor(matColor.R, matColor.G, matColor.B, 1.0f),
+                DiffuseColor = materialColor,
                 SpecularColor = PhongMaterials.ToColor(0.0225, 0.0225, 0.0225, 1.0),
                 EmissiveColor = PhongMaterials.ToColor(0.0, 0.0, 0.0, 1.0),
                 SpecularShininess = 12.8f,
             };
 
+            var selColor = (System.Windows.Media.Color)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["SelectionColor"];
+            selectionColor = new Color4(selColor.R/255.0f, selColor.G/255.0f, selColor.B/255.0f, selColor.A/255.0f);
+            SelectedMaterial = new PhongMaterial
+            {
+                Name = "White",
+                AmbientColor = PhongMaterials.ToColor(0.1, 0.1, 0.1, 1.0),
+                DiffuseColor = selectionColor,
+                SpecularColor = PhongMaterials.ToColor(0.0225, 0.0225, 0.0225, 1.0),
+                EmissiveColor = PhongMaterials.ToColor(0.0, 0.0, 0.0, 1.0),
+                SpecularShininess = 12.8f,
+            };
 
```

Comment:
This seems to exactly duplciate the material above.  Why not define it just once?